prometheus-to-sd/config/gce_config.go

```diff
@@ -17,10 +17,25 @@ limitations under the License.
 package config
 
 import (
+	"flag"
 	"fmt"
 	"strings"
 
 	gce "cloud.google.com/go/compute/metadata"
+	"github.com/golang/glog"
+)
+
+var (
```

Comment:
I just noticed that a convention for prom-to-sd is to include all flags in main.go: https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/prometheus-to-sd/main.go

Can you do the same for these? Here for example a zone parameter is passed, but it comes from zone-override flag.

Reply:
okay, done. I now use --node-name to override gce.InstanceName(), which I think is what you were suggesting.